### PR TITLE
Settings Panel Documentation

### PR DESCRIPTION
**Pull request by MWillWallT**
New page of control panel information.

Have taken the old information from the official documentation and updated it with all of the new + removed out of date elements.

### CODE REVIEW

File: Frontend/Docs/Settings Panel.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+### Notes
+
+**Quality Controller**
+Although Pixel Streaming adapts the quality of the stream to match the available bandwidth, the video frames are only encoded once by the Pixel Streaming Plugin. That one encoding is used for all clients. Therefore, only one client connection can "own" the quality used for adaptive streaming. If the other clients have a much better connection to the server, they may end up seeing a lower quality stream than necessary. On the other hand, if other clients have a much worse connection to the server, they may end up with lag or jitter.
+By default, each time a new browser connects, it adopts the ownership of the stream. Use this checkbox from any other connected browser to retake ownership.
+
+
+## Legal
+
+Copyright &copy; 2022, Epic Games. Licensed under the MIT License, see the file [LICENSE](./LICENSE) for details.
```
Comment (lukehb) [inline Frontend/Docs/Settings Panel.md]: Let's update copyright year to 2023

File: Frontend/Docs/Settings Panel.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+# Front End Setting Panel
+
+With the new Pixel Streaming Front End, we've created a settings panel that allows you to adjust and change settings of your stream on the fly.
```
Comment (lukehb) [inline Frontend/Docs/Settings Panel.md]: `frontend` instead of `Front End` - in this context it is understood to be correct usage

Comment (lukehb) [reply Frontend/Docs/Settings Panel.md]: It won't always be new.

I would prefer

> The Pixel Streaming frontend contains a settings panel that allows you to configure your Pixel Streaming user experience as needed - whether that be toggling features or connecting to different signalling servers.

File: Frontend/Docs/Settings Panel.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+# Front End Setting Panel
+
+With the new Pixel Streaming Front End, we've created a settings panel that allows you to adjust and change settings of your stream on the fly.
+This page will be updated with new features and commands available.
```
Comment (lukehb) [inline Frontend/Docs/Settings Panel.md]: > This page will be updated with new features and commands available.

to

> This page will be updated with new features and commands as they become available.

File: Frontend/Docs/Settings Panel.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+![Settings Panel](Resources/Images/settings-panel.png)
+
+### Pixel Streaming
+
+| **Setting** | **Description** |
+| --- | --- |
+| **Signalling URL** | The URL of the signalling server. |
+| **Streamer ID** | Allows you to select which streamer to stream. |
+| **Auto connect to stream** | Browser will automatically connect to the stream when loaded. Prevents having to click to start |
+| **Auto play video** | When stream is ready, starts playing video immediately instead of showing a play button. |
```
Comment (lukehb) [inline Frontend/Docs/Settings Panel.md]: > When stream is ready

to 

> When the stream is ready

File: Frontend/Docs/Settings Panel.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+| **Browser send offer** | The browser will start the WebRTC handshake instead of the Unreal Engine application. This is an advanced setting for people customising the front end. Primarily for backwards compatibility for 4.x versions of the engine. |
```
Comment (lukehb) [inline Frontend/Docs/Settings Panel.md]: people -> users

front end -> frontend

File: Frontend/Docs/Settings Panel.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+| **Use microphone** | Will start receiving audio input from your microphone and play it back through the stream. |
```
Comment (lukehb) [inline Frontend/Docs/Settings Panel.md]: Will start receiving audio input from your microphone and transmit it to the Unreal Engine.

File: Frontend/Docs/Settings Panel.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+| **Start video muted** | Muted audio when stream starts |
```
Comment (lukehb) [inline Frontend/Docs/Settings Panel.md]: when the stream starts

File: Frontend/Docs/Settings Panel.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+| **Prefer SFU** | Will attempt to use the Selective Forwarding Unit, will only work if you have one running. |
```
Comment (lukehb) [inline Frontend/Docs/Settings Panel.md]: > Will attempt to use the Selective Forwarding Unit, will only work if you have one running.

to

> Will attempt to use the Selective Forwarding Unit (SFU), if you have one running.

File: Frontend/Docs/Settings Panel.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+| **Force TURN** | Will attempt to connect exclusively via the TURN server. Will not work without an active TURN server. |
+| **Suppress browser keys** | Suppress or allow certain keys we use in UE, for example F5 to show shader complexity instead of refreshing the page. |
+| **AFK if Idle** | Timeout the connection if no input is detected for a period of time. |
+| **AFK timeout** | Allows you to specify the AFK timeout period. |
+
+
+### UI
+| **Setting** | **Description** |
+| --- | --- |
+| **Match viewport resolution** | Visualizes statistics about the connection between the browser and the Unreal Engine application. |
```
Comment (lukehb) [inline Frontend/Docs/Settings Panel.md]: This text is wrong. Should be:

> Resizes the Unreal Engine application resolution to match the browser's video element size.

File: Frontend/Docs/Settings Panel.md [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+| **Control scheme** | Will dictate if the stream captures your mouse or keeps it free. |
```
Comment (lukehb) [inline Frontend/Docs/Settings Panel.md]: If the scheme is `locked mouse` the browser will use `pointerlock` to capture your mouse, whereas if the scheme is `hovering mouse` you will retain your OS/browser cursor.

File: Frontend/Docs/Settings Panel.md [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+| **Streamer ID** | Allows you to select which streamer to stream. |
+| **Auto connect to stream** | Browser will automatically connect to the stream when loaded. Prevents having to click to start |
+| **Auto play video** | When stream is ready, starts playing video immediately instead of showing a play button. |
+| **Browser send offer** | The browser will start the WebRTC handshake instead of the Unreal Engine application. This is an advanced setting for people customising the front end. Primarily for backwards compatibility for 4.x versions of the engine. |
+| **Use microphone** | Will start receiving audio input from your microphone and play it back through the stream. |
+| **Start video muted** | Muted audio when stream starts |
+| **Prefer SFU** | Will attempt to use the Selective Forwarding Unit, will only work if you have one running. |
+| **Is quality controller?** | Makes the encoder of the Pixel Streaming Plugin use the current browser connection to determine the bandwidth available, and therefore the quality of the stream encoding. **See notes below** |
+| **Force mono audio** | Force browser to request mono audio in the SDP. |
```
Comment (lukehb) [inline Frontend/Docs/Settings Panel.md]: Force the browser

File: Frontend/Docs/Settings Panel.md [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+| **Min Bitrate (kbps)** | The minimum bitrate WebRTC should use. |
+| **Max Bitrate (kbps)** | The maximum bitrate WebRTC should use. |
+
+
+### Commands
+| **Setting** | **Description** |
+| --- | --- |
+| **Show FPS** | Will display the current FPS |
+| **Request Keyframe** | Will ask the stream for a keyframe. This is helpful if your stream is choppy and needs to catch up.  |
+| **Restart Stream** | Restarts the peer connection to the stream. Should be used if you change above settings to ensure they are applied. |
```
Comment (lukehb) [inline Frontend/Docs/Settings Panel.md]: Restarts the stream by disconnecting and reconnecting the websocket connection.

File: Frontend/Docs/Settings Panel.md [hunk 13]
```diff
@@ -0,0 +1,71 @@
...
+
+
+### Notes
+
+**Quality Controller**
+Although Pixel Streaming adapts the quality of the stream to match the available bandwidth, the video frames are only encoded once by the Pixel Streaming Plugin. That one encoding is used for all clients. Therefore, only one client connection can "own" the quality used for adaptive streaming. If the other clients have a much better connection to the server, they may end up seeing a lower quality stream than necessary. On the other hand, if other clients have a much worse connection to the server, they may end up with lag or jitter.
```
Comment (lukehb) [inline Frontend/Docs/Settings Panel.md]: match the available bandwidth, when using H.264 the video frames

....

they may end up with lag or jitter. Note, this quality controller setting is completely irrelevant if you are streaming using any codec other than H.264 (such as VP8 or VP9).

**Review by lukehb — CHANGES_REQUESTED** [review]
Left some comments

File: Frontend/Docs/Settings Panel.md (hunk 9 above)
Comment (MWillWallT) [reply Frontend/Docs/Settings Panel.md]: Ah! I did see this, missed fixing it sorry. Good catch.

**Comment by MWillWallT** [comment]
Made all proposed changes. Let me know if it's up to snuff!

**Review by lukehb — APPROVED** [review]
Looks good now!

**Pull request merged**